airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -86,6 +86,11 @@ public AirbyteConnectionStatus check(final JsonNode config) throws Exception {
       }
 
       return new AirbyteConnectionStatus().withStatus(Status.SUCCEEDED);
+    } catch (final ConfigErrorException configErrorException) {
```

Comment:
There isn't a need to catch and handle this exception here - it's handled at the top-level in https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/IntegrationRunner.java#L155